examples/Basics/Basics_Header.md

```diff
@@ -2,4 +2,4 @@
 
 These examples demonstrate the most basic aspects of BayBE: How to set up a
 {doc}`Campaign </userguide/campaigns>` and how to configure an optimization
-{doc}`Strategy </userguide/strategies>`.
\ No newline at end of file
+{doc}`Recommenders </userguide/recommenders>`.
```

Comment:
Would use singular here as the sentence is "How to configure an optimization recommender".

Reply:
fixed